sdk/eidolon_ai_sdk/builtins/logic_units/web_search.py

```diff
@@ -51,7 +51,7 @@ async def go_to_url(self, url: str) -> str:
         async with self._get(url=url) as resp:
             text = resp.text
             if not resp.is_success:
-                logger.warning(f"Request to url '{url}' return {resp.status}, {resp.reason}")
+                logger.warning(f"Request to url '{url}' return {resp.status_code}")
```

Comment:
unrelated, but I notice this was wrong from a recent test failure due to network issues.